generators/task-runner/index.js

```diff
@@ -0,0 +1,53 @@
+const yeoman = require('yeoman-generator');
+const yosay = require('yosay');
+const _merge = require('lodash/merge');
+const _includes = require('lodash/includes');
+
+module.exports = yeoman.Base.extend({
+  prompting: {
+    taskrunner() {
+      const self = this;
+
+      this.log(yosay(
+        'Configure Task Runner for Grunt'
+      ));
+      return this.prompt([{
+        type: 'checkbox',
+        name: 'selectedTaskRunner',
+        message: 'Which Task Runner do you use?',
+        choices: ['Grunt'],
+        default: ['Grunt'],
+        when: () => !self.config.get('selectedTaskRunner'),
+      }])
+      .then(answers => {
+        self.selectedTaskRunner = self.config.get('selectedTaskRunner') ||
+        answers.selectedTaskRunner;
+      });
+    },
+  },
+  writing() {
+    // Read a package.json if it exists or create it
+    const packageJSON = this.fs.readJSON(this.destinationPath('package.json'), {});
+    // Task Runner
+    if (_includes(this.selectedTaskRunner, 'Grunt')) {
+      this.log('Configuring Gruntfile for Task Runner');
+      _merge(packageJSON, {
+        devDependencies: {
+          grunt: '^1.0.1',
+          'load-grunt-config': '^0.19.2',
+        },
+      });
+      this.fs.copyTpl(
+        this.templatePath('grunt/aliases.js'),
+        this.destinationPath('./grunt/aliases.js'), {}
+      );
+      this.fs.copyTpl(
```

Comment:
@riteshe63 This should be first then aliases and how about a `this.log` with `Creating a Gruntfile` and `Creating an alias file for grunt`.